stable/selenium/templates/chromeDebug-deployment.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
 spec:
   replicas: {{ .Values.chromeDebug.replicas }}
+  selector:
+    matchLabels:
+      chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
this will break on chart-upgrades?

Reply:
I couldn't install first without this, How can I test upgrade without installing it? Please help with ideas, I will follow.

Reply:
@davidkarlsen, You are correct. Having [version number and release date will break](https://helm.sh/docs/chart_best_practices/#podtemplates-should-declare-selectors) the upgrade. I am fixing it with with app, release labels only. Let me know if you have further comment.

Reply:
@davidkarlsen Please review the latest changes.